src/Display/Tabs/Tabs.tsx

```diff
@@ -111,12 +111,14 @@ type Tabs = React.FunctionComponent<Props> & {
 
 Tabs.Pane = TabPane;
 
-interface Props {
+export interface Props {
   children: React.ReactNode;
   variant?: Variant;
```

Comment:
I think `variant` only works for `alignment` prop is `horizontal`. So probably we should have something like `{ alignment: 'vertical', variant: never } | {alignment: 'horizontal',  variant: Variant}` to type check it. Btw, haven't checked if it works. 😅

Reply:
How would I add this type check? Oo

Reply:
I think below would work. 🙏 
```
type CommonProps = {
  children: React.ReactNode;
  theme?: Theme;
  activeTab?: string | number;
  onTabClick?(tab: React.ReactText | React.ReactNode): void;
  className?: string;
};
type Props =
  | CommonProps & {
      alignment: 'vertical';
      variant: never;
    }
  | {
      alignment: 'horizontal';
      variant: Variant;
    };

```

Reply:
This seems to work, but I'm getting an error `Property 'variant' does not exist on type 'PropsWithChildren<Props>'.` because of the default variant.

Let's just keep it the way it is right now, we've got a redesign of the tabs in the pipeline anyway that will remove the variants.